cumulusci/robotframework/Salesforce.robot

```diff
@@ -88,9 +88,14 @@ Open Test Browser
     ...  The keyword `Log Browser Capabilities` will automatically be called.
     ...  The keyword will also call `Wait Until Salesforce is Ready` unless
     ...  the `wait` parameter is set to False.
+    ...
+    ...  The arguments 'username' or 'useralias' may be used to specify
+    ...  a specific user; if not specified then the org default will be used.
+
+    [Arguments]  ${size}=${DEFAULT BROWSER SIZE}  ${alias}=${NONE}  ${wait}=True  ${useralias}=${NONE}
```

Comment:
I'm a little confused between the `alias` and `useralias` arguments here. `useralias` is passed to `Login URL`, and `alias` is passed to `Open Browser`. Do we need both?

Reply:
The `alias` argument  has been there for a while, and is an alias for the browser instance. I would prefer that `alias` was `browseralias` but I didn't want to change it for backward-compatibility reasons.

Reply:
Okay, that makes sense. Can you add `alias` to the docstring here to clarify the difference?